--- a/chktex-1.7.10/ChkTeX.h
+++ b/chktex-1.7.10/ChkTeX.h
@@ -46,6 +46,11 @@
 #include <ctype.h>
 #include <math.h>
 #include <stdio.h>
+
+#ifdef _MSC_VER
+#  include <direct.h>
+#  include <io.h>
+#endif
 
 #ifdef HAVE_STDARG_H
 #  include <stdarg.h>
--- a/chktex-1.7.10/Utility.h
+++ b/chktex-1.7.10/Utility.h
@@ -166,7 +166,7 @@
 
 
 #ifndef HAVE_STRCASECMP
-int strcasecmp(char *a, char *b);
+int strcasecmp(const char *a, const char *b);
 #endif
 
 #if !(defined HAVE_DECL_STPCPY && HAVE_DECL_STPCPY)
